nilearn/maskers/nifti_labels_masker.py

```diff
@@ -58,10 +59,12 @@ class NiftiLabelsMasker(BaseMasker, _utils.CacheMixin):
         Region definitions, as one image of labels.
 
     labels : :obj:`list` of :obj:`str`, optional
```

Comment:
the input type of labels is never checked

this was not a big issue for version < 0.10.3 as those were only used when generating masker reports, but this can now cause trouble

we should probably update the doc string to say that we can accept sequence of strings

though this may not be enough as some nilearn atlases have their labels in nump arrays